djangorestapi/api/tests.py

```diff
@@ -54,3 +54,12 @@ def test_empty_whitespace(self):
     def test_valid_data(self):
         self.assertEqual(self.response.status_code,201)
 
+    def test_fetch_all_todo_items(self):
```

Comment:
I think You meant Fetch one in your get all test

Reply:
Noted